util/HighLevelFeatures.py

```diff
@@ -129,6 +129,8 @@ def _DrawSingleLayer(self, data, layer_nr, filename, title=None, fig=None, subpl
         ax.grid(False)
         if vmax is None:
             vmax = data.max()
+            vmax = max(1e-1, vmax)
```

Comment:
Can you print a warning message here to make sure we can debug plot issues easily please

Reply:
Done